src/media/js/routes.js

```diff
@@ -34,6 +34,11 @@ define('routes',
         {'pattern': '^/usage$', 'view_name': 'usage'},
     ]);
 
+    var search = '/api/v2/fireplace/search/?cache=1&vary=0';
+    if (settings.meowEnabled) {
+        search = '/api/v2/multi-search/?cache=1&vary=0';
```

Comment:
Yeah, we'd need 3 API endpoints otherwise. Just apps, just websites and a combined one. I think it makes sense to just have the combined one allow filtering.

For that matter, maybe the content type should default to apps and we can just use the regular search endpoint? When `settings.meowEnabled` we can send `content=all`.